### ISSUE TITLE
Usage with Nuxt and Vuex

### ISSUE THREAD

**Issue by uuf6429** [vue-styleguidist/vue-styleguidist#1313]
I've spent a disproportionate amount of time trying to set this up.

The problem I have is that the nuxt example is actually more like slapping nuxt on top of the vuex example.

Nuxt takes care of automatically loading vuex files, but they have to have a particular structure.
Meaning that while the example works, no one is going to manually write a file to load all store modules and/or the additional boilerplate of actually exporting a vuex module.

You can see the differences between [Vuex](https://vuex.vuejs.org/guide/structure.html) and [Nuxt](https://nuxtjs.org/docs/directory-structure/store/):
- vuex: index file to (manually) load other modules/files
- nuxt: index file is the main namespace (but otherwise contains code like other store file)
- vuex: module must have a default export (`export const mutations = {};  export default { mutations }`)
- nuxt: module does not need a default export (`export const mutations = {}`)

**TLDR:** I'm looking for a way to load the vuex store of a nuxt-based project without having to manually add each store module manually and having to make the modules compatible with traditional vuex.

**Comment by elevatebart** [comment]
Hello @uuf6429 ,

Sorry to hear you had to spend so long setting all of this up.

Would you mind creating a simple repository with `nuxt` and `vuex` setup as they are in your project?
Adding vue-styleguidist to the project and pointing out what does not seem to work would help tremendously as well.

I know I am asking a lot, but it will take me significantly longer to get to where you are that for you to share it with me.

Thank you in advance
